gitapp/lib/users_list/no_connection_widget.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:gitapp/constants/constants.dart';
+
+class networkErrorView extends StatelessWidget {
```

Comment:
As above. We should name classes from Uppercase

Reply:
Solved